demo/universal/native/lib/ServerFunctions.re

```diff
@@ -309,8 +347,74 @@ module Samples = {
       )
     };
 
+  // Lets say this is the server action declared by the end-user
+  /**
+    [@react.server.action]
+    let simpleResponse = () => {
+      Lwt.return(
+        (`String("Hello from server with simple response action")),
+      );
+    };
+  */
+  // It's going to be on top to this that we are going to generate the codes bellow
+  let errorId = "id/samples/error";
+
+  [@platform native]
+  let error = () => {
+    let stack = createStackError();
+
+    failwith(Printf.sprintf("Error from server"));
+  };
+
+  // This is the router  handler that will handle parsing the args and calling the handler the user declared
+  // This code will be generated by the ppx automatically
+  [@platform native]
+  let errorRouteHandler = _args =>
+    try(error() |> Lwt.map(response => response)) {
+    | e =>
+      let msg = Printexc.to_string(e);
+      Lwt.return(
+        React.Error({
+          message: msg,
+          stack: createStackError(),
```

Comment:
This stack will be from server. Is there a way to show the server(.re) files on the browser?